### PR TITLE
Implement unit tests for business logic

### PR DESCRIPTION
**Pull request by DariaSova**
Test business logic
Test edge cases and errors

### CODE REVIEW

File: account.go [hunk 1]
```diff
@@ -3,8 +3,9 @@ package main
 import (
 	"time"
 
+	"./src/utils"
```
Comment (DariaSova) [inline account.go:6]: you don't need to specify path for packages inside src folder
